pkgs/top-level/python-packages.nix

```diff
@@ -4660,6 +4660,11 @@ in {
     inherit (pkgs) pkgconfig;
   };
 
+  wxPython_4_0 = callPackage ../development/python-modules/wxPython/4.0.nix {
```

Comment:
I think this should either follow the naming convention of `wxPython30`, or `wxPython30` should be renamed (with an alias for compatibility).

Reply:
Also, since wxPython 3.0 does not support Python 3, we might want to make the `wxPython` package point to 4.0 on Python 3. Even if some packages aren't compatible, it seems better than having every package that uses wxPython fail to evaluate on Python 3.

Reply:
Is WxPython 4 API compatible with 3?

Reply:
They are [not fully API compatible](https://wxpython.org/Phoenix/docs/html/MigrationGuide.html). Any application that wants to target both Python 2 and 3 is probably going to use wxPython 4.0, but it might be possible to be compatible with both 3.0 and 4.0.

Making 4.0 the default for Python 3 will only make packages that currently fail to evaluate, fail to build.

Reply:
@lopsided98 see https://github.com/NixOS/nixpkgs/pull/64047#discussion_r299567157

I agree re pointing wxPython to 4.0 for python3..do you know how to do this? I’m familiar with overrides but not with how to do this in nixpkgs